e2e/tests/courtAdminMangesHearings_test.js

```diff
@@ -0,0 +1,307 @@
+const config = require('../config.js');
+const hearingDetails = require('../fixtures/hearingTypeDetails.js');
+const dateFormat = require('dateformat');
+const dateToString = require('../helpers/date_to_string_helper');
+const mandatoryWithMultipleChildren = require('../fixtures/caseData/mandatoryWithMultipleChildren.json');
+const supportingEvidenceDocuments = require('../fixtures/supportingEvidenceDocuments.js');
+const moment = require('moment');
+
+let caseId;
+let submittedAt;
+let hearingStartDate;
+let hearingEndDate;
+
+Feature('Hearing administration');
+
+BeforeSuite(async ({I}) => {
+  caseId = await I.submitNewCaseWithData(mandatoryWithMultipleChildren);
+  submittedAt = new Date();
+
+  await I.navigateToCaseDetailsAs(config.hmctsAdminUser, caseId);
+});
+
+Before(async ({I}) => await I.navigateToCaseDetailsAs(config.hmctsAdminUser, caseId));
+
+Scenario('HMCTS admin enters FamilyMan case number', async ({I, caseViewPage, enterFamilyManCaseNumberEventPage}) => {
```

Comment:
Do we want this to be part of the before all?

Reply:
done